src/createServer.js

```diff
@@ -1,3 +1,31 @@
-// Write code here
-// Also, you can create additional files in the src folder
-// and import (require) them here
+const http = require('http');
+const { convertToCase } = require('./convertToCase');
+const { handleError } = require('./errorMessages');
+
+function createServer() {
+  const getServer = http.createServer((request, response) => {
+    response.setHeader('Content-type', 'application/json');
+
+    const updatedUrl = new URL(request.url, `http://${request.headers.host}`);
+    const textToConvert = updatedUrl.pathname.slice(1) || '';
+    const caseToConvert = updatedUrl.searchParams.get('toCase') || '';
```

Comment:
If am not mistaken there is no need || because in case of apsent params, there are already empty strings